docs/ingest/configuration/multi-level-partitioned-tables.mdx

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Do we need this file? If you Cmd/Ctrl F for "Hive" in our documentation, you'll find we have documentation around Hive partitioning under CSV, JSON, Parquet, etc.

If we need to update anything for Hive partitioning with the support you are adding, could we please do it there? I don't think this should be free-floating in a separate page.